industrial_extrinsic_cal/CMakeLists.txt

```diff
@@ -1,229 +1,243 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(industrial_extrinsic_cal)
 
-## Find catkin macros and libraries
-## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
-## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS roscpp std_msgs cv_bridge tf roslint std_srvs roslib genmsg actionlib_msgs actionlib moveit_ros_planning_interface geometry_msgs sensor_msgs)
-
-# Opencv
-FIND_PACKAGE(OpenCV REQUIRED)
-IF (OPENCV_FOUND)
-  MESSAGE("-- Found OpenCV version ${OPENCV_VERSION}: ${OPENCV_INCLUDE_DIRS}")
-ENDIF (OPENCV_FOUND)
-
-# Ceres
-FIND_PACKAGE(Ceres REQUIRED)
-IF (CERES_FOUND)
-  MESSAGE("-- Found Ceres version ${CERES_VERSION}: ${CERES_INCLUDE_DIRS}")
-ENDIF (CERES_FOUND)
-
-SET (CMAKE_CXX_FLAGS -fPIC)
-
-# Eigen
-#FIND_PACKAGE(Eigen REQUIRED)
-#IF (EIGEN_FOUND)
-#  MESSAGE("-- Found Eigen version ${EIGEN_VERSION}: ${EIGEN_INCLUDE_DIRS}")
-#ENDIF (EIGEN_FOUND)
-
-## System dependencies are found with CMake's conventions
-# find_package(Boost REQUIRED COMPONENTS system)
-
-## Uncomment this if the package has a setup.py. This macro ensures
-## modules and global scripts declared therein get installed
-## See http://ros.org/doc/groovy/api/catkin/html/user_guide/setup_dot_py.html
-# catkin_python_setup()
-
-#######################################
-## Declare ROS messages and services ##
-#######################################
-
-## Generate messages in the 'msg' folder
-# add_message_files(
-#   FILES
-#   Message1.msg
-#   Message2.msg
-# )
-
-## Generate services in the 'srv' folder
- add_service_files(
-   FILES
-   get_mutable_joint_states.srv
-   set_mutable_joint_states.srv
-   store_mutable_joint_states.srv
-   camera_observer_trigger.srv
-   user_accept.srv
-   calibrate.srv
-   covariance.srv
- )
-
-add_action_files(DIRECTORY  action  FILES  manual_trigger.action robot_joint_values_trigger.action  robot_pose_trigger.action manual_target_location.action calibration.action)
-
-## Generate added messages and services with any dependencies listed here
-# generate_messages(
-#   DEPENDENCIES
-#   std_msgs  # Or other packages containing msgs
-# )
+find_package(catkin REQUIRED COMPONENTS
```

Comment:
In case I forget to thank you for all this, Thank you very much.  I don't have a good handle on CMakeList syntax.